test/dummy/app/components/card/card.js

```diff
@@ -0,0 +1 @@
+console && console.log('card')
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'console' is not defined.